Lib9c/Action/Buy8.cs

```diff
@@ -0,0 +1,323 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Bencodex.Types;
+using Lib9c.Model.Order;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Assets;
+using Nekoyume.Model.EnumType;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.Mail;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using Serilog;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
+    [ActionObsolete(2100000)]
```

Comment:
We need to change this value (and others in this PR too).

Reply:
hmm.. `2120000` Would be okay?

Reply:
Considering the release schedule, I think 2.2m (2,200,000) would be fine.

Reply:
@longfin I changed this number (and others in this PR too)